### PR TITLE
Allow view of points to construct SimpleMesh

### PR DESCRIPTION
**Pull request by deltaeecs**
Fix https://github.com/JuliaGeometry/Meshes.jl/issues/187 by employing  @juliohm's suggestion at https://github.com/JuliaGeometry/Meshes.jl/issues/187#issuecomment-959588059

### CODE REVIEW

File: test/mesh.jl [hunk 1]
```diff
@@ -186,6 +186,11 @@
     topo = topology(mesh)
     @test collect(faces(topo, 2)) == [connect((3,4,1), Triangle)]
     @test collect(faces(topo, 3)) == [connect((1,2,3,4), Tetrahedron)]
+    
+    # test for https://github.com/JuliaGeometry/Meshes.jl/issues/187
```
Comment (juliohm) [inline test/mesh.jl:190]:
```suggestion
    # test for https://github.com/JuliaGeometry/Meshes.jl/issues/187
    points = P3[(0,0,0),(1,0,0),(1,1,1),(0,1,0)]
```

File: test/mesh.jl [hunk 2]
```diff
@@ -186,6 +186,12 @@
...
+    points = P3[(0,0,0),(1,0,0),(1,1,1),(0,1,0)]
+    tetra  = Tetrahedron(view(points, 4:-1:1))
+    facets = boundary(tetra)
```
Comment (juliohm) [inline test/mesh.jl]: Please refactor this test so that it is just a simple call to the SimpleMesh constructor. That was the source of the issue. 👌🏽

Comment (deltaeecs) [reply test/mesh.jl]: ok

**Comment by juliohm** [comment]
@serenity4 do you have any idea why the codecov bot is consistently producing this failure in PRs? This PR is completely unrelated to the coverage changes in FIST.

**Comment by serenity4** [comment]
> @serenity4 do you have any idea why the codecov bot is consistently producing this failure in PRs? This PR is completely unrelated to the coverage changes in FIST.

Nope, no idea on my side unfortunately.

File: test/mesh.jl (hunk 2 above)
Comment (juliohm) [reply test/mesh.jl]: @deltaeecs you can keep pushing changes to this branch to update it. You don't need to start a fresh PR from a fresh branch. Let me know if you need some help with that.

Comment (deltaeecs) [reply test/mesh.jl]: I change the test file, is it ok?

**Comment by juliohm** [comment]
Thanks for this contribution! 🎉

**Pull request merged**